plugins/chat/assets/javascripts/discourse/lib/chat-live-panel.js

```diff
@@ -0,0 +1,26 @@
+import { tracked } from "@glimmer/tracking";
+import { action } from "@ember/object";
+import { TrackedArray } from "@ember-compat/tracked-built-ins";
+
+export default class ChatLivePanel {
+  @tracked messages = new TrackedArray();
+
+  @tracked selectingMessages;
+  @tracked lastSelectedMessage;
+
+  onSelectMessage(message) {
```

Comment:
Naming for all the panel stuff should match up with the chat message actions IMO with an on- prefix, e.g. onSelectMessage, onDeleteMessage etc.